.github/workflows/deploy_api.yml

```diff
@@ -9,7 +9,7 @@ on:
   schedule:
    - cron: '30 0,12 * * *'
 
-  # push:
+  push:
```

Comment:
Was this not enabled before for a particular reason?

Reply:
it's enabled just for testing.  I take it off before.

We can't have an hour long job go off on every push.